src/test/java/uk/gov/pay/ledger/queue/QueueMessageReceiverIT.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.pay.ledger.it.queue.sqs;
+package uk.gov.pay.ledger.queue;
```

Comment:
[Maybe] `QueueMessageReceiver` is under `uk.gov.pay.ledger.queue.managed` so maybe this test should go there too.